src/forklift/messaging.py

```diff
@@ -9,43 +9,62 @@
 import logging
 import secrets
 from config import get_config_prop
+from email import encoders
+from email import generator
+from email.mime.base import MIMEBase
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
+from os.path import isfile
+from os.path import join
 from smtplib import SMTP
 
 
 log = logging.getLogger('forklift')
 
 
-def send_email(to, subject, body):
+def send_email(to, subject, body, attachment, save_to_file=False):
     '''
     to: string | string[]
     subject: string
     body: string | MIMEMultipart
 
     Send an email.
     '''
-
     if not isinstance(to, basestring):
         to_addresses = ','.join(to)
     else:
         to_addresses = to
 
     if isinstance(body, basestring):
         message = MIMEMultipart()
-        message.attach(MIMEText(body))
+        message.attach(MIMEText(body, 'html'))
     else:
         message = body
 
     message['Subject'] = subject
     message['From'] = secrets.from_address
     message['To'] = to_addresses
 
-    smtp = SMTP(secrets.smtp_server, secrets.smtp_port)
-    if get_config_prop('sendEmails'):
+    if isfile(attachment):
+        log_file_attachment = MIMEBase('application', 'octet-stream')
+        log_file_attachment.add_header('Content-Disposition', 'attachment; filename="forklift.log"')
```

Comment:
Can value for `filename` dynamically come from `attachment` to allow for any file to be attached to an email rather than hardcoding `forklift.log` as the name.